app/services/bill-runs/submit-cancel-bill-run.service.js

```diff
@@ -48,15 +46,7 @@ async function go (billRunId) {
   }
 
   await _updateStatusToCancel(billRunId)
-
-  // NOTE: We originally believed that should anything error in _cancelBillRun() the try/catch in the controller would
-  // catch it. However, when testing this theory we found it crashed the test framework. So, we tried it for real and
-  // again confirmed we'd crash the service. The controller would never see the error. It is because we are not awaiting
-  // this call that any errors thrown are considered uncaught. However, if we instead use the ES6 variant the error _is_
-  // caught. All we can do at this point is log it.
-  _cancelBillRun(billRun).catch((error) => {
-    global.GlobalNotifier.omfg(error.message, { billRunId }, error)
-  })
+  await _cancelBillRun(billRun)
```

Comment:
Just want to check why the error handling has been removed. 

As i read it we no longer require this specific error message to be used. There for the error will just bubble up.

Reply:
Because it is already handled in the controller that uses this service, and the intent was always to get the error to bubble up to there (it wasn't, hence the old-style promise handling).

However, I've spotted something I've broken (related to this 😁 ), so taking it back to update. Thanks for looking through it though.

Reply:
Resolve when you are happy. I can reapprove if needed.